wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/CreatePasswordSecuredLink/CreateSecureGuestLinkViewModel.swift

```diff
@@ -46,6 +47,21 @@ final class CreateSecureGuestLinkViewModel {
         delegate?.viewModel(self, didGeneratePassword: randomPassword)
     }
 
+    func validatePassword(
+        for textField: ValidatedTextField,
+        against confirmPasswordField: ValidatedTextField
+    ) -> Bool {
+
+        guard let updatedString = textField.text,
```

Comment:
[Opinion] would this be clearer?
```suggestion
        return textField.text != nil &&
    confirmPasswordField.text != nil &&
    !textField.text!.isEmpty &&
    textField.text == confirmPasswordField.text &&
    textField.isValid
```

Reply:
Or
```suggestion
        guard let enteredPassword = textField.text, 
            let confimationText = confirmPasswordField.text,
            !enteredPassword.isEmpty,
            textField.isValid,
            enteredPassword == confimationText else {
        return false
    }
    return true
```

Reply:
Done!